catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -207,9 +210,14 @@ private ContentItem createDerivedImage(String id, String qualifier, BufferedImag
         return null;
     }
 
-    private String buildDerivedImageTitle(String title, String qualifier) {
+    protected String buildDerivedImageTitle(String title, String qualifier) {
         String rootFileName = FilenameUtils.getBaseName(title);
-        return String.format(DERIVED_IMAGE_FILENAME_PATTERN, qualifier, rootFileName, JPG);
+
+        return (StringUtils.isNotBlank(rootFileName) &&
```

Comment:
Ah yes. I see that. I guess my only quibble then would be with the ternary. With that long/complex a conditional, I think it's a little harder to read than with an if/else (and you could remove the else and just return at the end).